mkdocs.yml

```diff
@@ -12,6 +12,7 @@ pages:
     - Migration:
       - 'Version 2 to Version 3': migration/v2-to-v3.md
 site_name: zend-mvc-console
-site_description: Zend\Mvc\Console
+site_description: 'Integration between zend-mvc and zend-console.'
 repo_url: 'https://github.com/zendframework/zend-mvc-console'
-copyright: 'Copyright (c) 2005-2018 <a href="https://www.zend.com/">Zend Technologies USA Inc.</a>'
+extra:
+    show_special_homepage: True
```

Comment:
should we use lowecase `true` and also missing empty line at the end of the file

Reply:
> should we use lowercase `true`

Is there a coding standard for YAML and for MkDocs / Jinja / Python it does not matter. 😉